modules/ROOT/pages/administration/access-control/database-administration.adoc

```diff
@@ -182,6 +182,10 @@ REVOKE [IMMUTABLE] database-privilege ON { HOME DATABASE \| DATABASE[S] { * \| n
 Use `REVOKE` if you want to remove a privilege.
 ====
 
+Common errors such as misspellings or attempting to grant privileges to roles where either does not exist will lead to notifications.
+Some of these notifications may be replaced with errors in a future major version of neo4j.
```

Comment:
Either we only mention the cases that'll give errors or we also mention the cases that'll stay notifications, right now it feels like we have a combo with this sentence (giving only the error cases and then saying some as if not all error cases will be throwing errors later)

Reply:
I think we could leave this non-specific. As long as the details are described on the notifications page.